### PR TITLE
Removed extraneous columns and updated data.mode to a string.

### PR DESCRIPTION
**Pull request by achasmita**

### CODE REVIEW

**Comment by achasmita** [comment]
I have excluded following columns:
``` bash
EXCLUDED_TRAJECTORIES_COLS = [
    'data.loc.type',
    'data.loc.coordinates',
    'data.local_dt.year',
    'data.local_dt.month',
    'data.local_dt.day',
    'data.local_dt.hour',
    'data.local_dt.minute',
    'data.local_dt.second',
    'data.local_dt.weekday',
    'data.local_dt.timezone',
    'data.local_dt_year',
    'data.local_dt_month',
    'data.local_dt_day',
    'data.local_dt_hour',
    'data.local_dt_minute',
    'data.local_dt_second',
    'data.local_dt_weekday',
    'data.local_dt_timezone',
]
```

**Comment by achasmita** [comment]
Here is the screenshot for columns in recent table:
Screen Shot 2023-10-03 at 11 01 30 AM
Screen Shot 2023-10-03 at 11 02 32 AM
Screen Shot 2023-10-03 at 11 02 57 AM

File: utils/db_utils.py [hunk 1]
```diff
@@ -140,6 +142,10 @@ def query_trajectories(start_date, end_date):
     df.drop(columns= columns_to_drop, inplace=True) 
     modified_columns = perm_utils.get_trajectories_columns(df.columns)  
     df.columns = modified_columns 
+    for col in constants.EXCLUDED_TRAJECTORIES_COLS:
+        if col in df.columns:
+            df.drop(columns= [col], inplace=True) 
+    df['data.mode'] = df['data.mode'].apply(lambda x: ecwm.PredictedModeTypes(x).name if x in set(enum.value for enum in ecwm.PredictedModeTypes) else 'UNKNOWN')
```
Comment (shankari) [inline utils/db_utils.py:148]: `mode` is a `MotionTypes` not a `PredictedModeTypes`
Please make sure to understand and double-check the OpenPATH data model
https://github.com/e-mission/e-mission-server/blob/bc78230d3196ca10249b6e95898f63fddf924f6f/emission/core/wrapper/recreatedlocation.py#L20

**Review by shankari — CHANGES_REQUESTED** [review]

**Comment by shankari** [comment]
I can't modify this repo, so merging and then fixing immediately

**Review by shankari — APPROVED** [review]

**Pull request merged**